resources/views/auth/newLogin.blade.php

```diff
@@ -25,7 +26,14 @@
           @endphp
           @if (!$isMobile)
             <div class="slogan">
-              <img src="/img/slogan.svg" alt="ProcessMaker" />
+              <img src="/img/slogan1.svg" alt="ProcessMaker" />
+              <div class="typewriter-container d-flex align-items-center my-5">
+                <img src="/img/proceC2.svg" class="mr-2 procesC2-icon" alt="ProceC2" />
```

Comment:
Consider to change the `alt` content to a simple description, could be `ProcessMaker Bot`